packages/obonode/obojobo-chunks-list/editor-component.scss

```diff
@@ -1,6 +1,19 @@
 @import '~styles/includes';
 
 .obojobo-draft--chunks--list {
+	
+	.li-spacing-moderate {
```

Comment:
Better if these were more specific, so `&.li-spacing-moderate` - That ensures that the rule only applies on the root `obojobo-draft--chunks--list` element. Same comment for the other style.